metabase/metabase
# Show inactive tables in search results & recent items

Epic: https://github.com/metabase/metabase/issues/18675

<img width="589" alt="Screen Shot 2021-10-28 at 7 02 07 pm" src="https://user-images.githubusercontent.com/8542534/139320837-6492ec3c-ac56-4973-b341-4c58df930185.png">
<img width="529" alt="Screen Shot 2021-10-28 at 10 13 12 pm" src="https://user-images.githubusercontent.com/8542534/139320844-8cc14f96-3c30-4352-b565-e2ba39ee742e.png">
<img width="963" alt="Screen Shot 2021-10-28 at 10 13 17 pm" src="https://user-images.githubusercontent.com/8542534/139320846-bcb4759c-2293-45ed-8f72-93939af94557.png">

**How to test**
As the BE for this PR is not ready we would need to rely on mocked data:
1. Set `initial_sync` to false https://github.com/metabase/metabase/blob/2488618db0322577f03d4295d38d1ecbe327f64d/frontend/src/metabase/schema.js#L126
2. Set `initial_sync` to false https://github.com/metabase/metabase/blob/a96eebfda7cd8c04f1679a983d2ba2f319a68bb6/frontend/src/metabase/entities/search.js#L70
3. Navigate to `http://localhost:3000/`
4. Open search. It should display recent tables in disabled & loading state
5. Type table name. It should display found tables in disabled & loading state
6. Check that BE is polled for new statuses
7. Check that when all tables have `initial_sync: true`, BE is not polled

Please note that it should be OK to merge this PR before BE changes as with `initial_sync` set statically to `true` users won't see FE changes.
